File: square/src/main/java/com/raxdenstudios/square/interceptor/ApplicationSimpleInterceptor.java
package com.raxdenstudios.square.interceptor;

import android.app.Application;

/**
 * Created by Ángel Gómez
 *
 * This abstract class defines the basis of an application interceptor.
 */
public abstract class ApplicationSimpleInterceptor
        extends ApplicationInterceptor<Interactor, InterceptorCallback<Interactor>> {

    public ApplicationSimpleInterceptor(Application application) {
        super(application, null);
    }

}

File: square/src/main/java/com/raxdenstudios/square/interceptor/ApplicationSimpleInterceptor.kt
package com.raxdenstudios.square.interceptor

import android.app.Application

/**
 * Created by Ángel Gómez
 *
 * This abstract class defines the basis of an application interceptor.
 */
abstract class ApplicationSimpleInterceptor(
        application: Application)
    : ApplicationInterceptor<InterceptorCallback>(application)
